packages/financial-templates-lib/src/price-feed/TraderMadePriceFeed.js

```diff
@@ -30,7 +30,7 @@ class TraderMadePriceFeed extends PriceFeedInterface {
     getTime,
     minTimeBetweenUpdates,
     priceFeedDecimals = 18,
-    ohlcPeriod = 10 // Only 5, 10, 15 minutes is supported by TraderMade.
+    ohlcPeriod = 1 // Some pairs like CNYUSD are only available at higher granularities like 10
```

Comment:
is there any way to sanitise this or catch if it is misconfigured?

Reply:
Yes I added these lines to top of constructor:

```
    // Sanitize API specific parameters:
    assert(VALID_OHLC_PERIODS.includes(ohlcPeriod), `ohlcPeriod must be one of ${JSON.stringify(VALID_OHLC_PERIODS)}`);
    assert(minuteLookback <= MAX_MINUTE_LOOKBACK, `minuteLookback must be < ${MAX_MINUTE_LOOKBACK}`);
    assert(hourlyLookback <= MAX_HOURLY_LOOKBACK, `hourlyLookback must be < ${MAX_HOURLY_LOOKBACK}`);

```

where:

```
const VALID_OHLC_PERIODS = [1, 5, 10, 15, 30];
const MAX_MINUTE_LOOKBACK = 172800; // 2 days
const MAX_HOURLY_LOOKBACK = 5184000; //2 months
```